lib/src/generated/bledata.pbenum.dart

```diff
@@ -3,4 +3,4 @@
 //  source: bledata.proto
 //
 // @dart = 2.3
-// ignore_for_file: camel_case_types,non_constant_identifier_names,library_prefixes,unused_import,unused_shown_name,return_of_invalid_type
+// ignore_for_file: annotate_overrides,camel_case_types,unnecessary_const,non_constant_identifier_names,library_prefixes,unused_import,unused_shown_name,return_of_invalid_type,unnecessary_this,prefer_final_fields
```

Comment:
So, this is all it takes? No changes in Kotlin and Swift files?

Reply:
the issue was only related to dart code (some violation of pedantic) because dart protobuf deprecated the `copywith` method for performance reasons